docs/getting-started/ory-cloud-oauth2.mdx

```diff
@@ -0,0 +1,228 @@
+
+By following this guide you will:
+
+- Create an OAuth2 client in Ory Cloud.
+- Run a simple login and consent Node.js application that allows the user to grant or reject access to their data on your machine.
+- Configure Ory Cloud to work with the sample application that runs on your machine.
+- Perform Authorization Code Grant in Ory Cloud using the login and consent app and a sample web server acting as an OAuth2
+  Client.
+
+## Prerequisites
+
+Before you start, prepare your environment:
+
+- [Install Ory CLI](../guides/cli/installation).
+- Create an Ory Cloud account and project. Use [Ory CLI](../guides/cli/cli-basics#create-ory-cloud-project) or go to
+  [console.ory.sh](https://console.ory.sh/).
+- [Install Ory Hydra CLI](../hydra/install).
+- [Create a Personal Access Token (PAT)](../concepts/personal-access-token) and save it in a safe place for later use.
+
+## Setup
+
+### Create OAuth2 client in Ory Cloud
+
+Create a new OAuth2 client in Ory Cloud by sending a POST request to the `/admin/clients` API. This is a protected endpoint, which
+means that you must include the Ory PAT in the `Authorization` header of the request.
+
+Alternatively, you can use the Ory Hydra CLI and pass your Ory Cloud project slug and the Ory PAT using flags.
+
+:::tip
+
+Learn more about creating OAuth2 clients in Ory Cloud by reading the
+[API specification](../reference/api#operation/adminCreateOAuth2Client).
+
+:::
+
+````mdx-code-block
+import CodeBlock from '@theme/CodeBlock'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+  <TabItem value="curl" label="cURL" default>
+
+```bash
+curl -X POST 'SDK_CONFIGURATION_URL/admin/clients' \  # Found in the 'Connect' section of the Cloud Console. Must end with '/admin/clients'.
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer ORY_PAT' \  # Ory PAT copied from the Cloud Console, 'ory_pat' prefix included.
+  -d '{
+    "client_name": "YOUR_CLIENT_NAME",
+    "grant_types": [
+      "authorization_code"
+    ],
+    "redirect_uris": [
+      "http://127.0.0.1:5555/callback"
+    ],
+    "response_types": [
+          "code",
+          "id_token",
+          "refresh_token"
+    ],
+    "scope": "openid offline"
+  }'
+```
+
+  </TabItem>
+  <TabItem value="hydracli" label="Ory Hydra CLI">
+
+```shell
+hydra clients create \
+    --name YOUR_CLIENT_NAME \
+    --grant-types authorization_code,refresh_token \ # Mandatory for this example, don't change.
+    --response-types code,id_token \ # Mandatory for this example, don't change.
+    --scope openid,offline \ # Mandatory for this example, don't change.
+    --access-token ORY_PAT \ # Ory PAT copied from the Cloud Console, 'ory_pat' prefix included.
+    --endpoint https://SDK_CONFIGURATION_URL/admin # Found in the 'Connect' section of the Cloud Console. Must end with '/admin'.
+    --callbacks http://127.0.0.1:5555/callback \ # Mandatory for this example, don't change.
```

Comment:
Trailing `\` means that there is another line coming :) If it is the last line, omit it:

```suggestion
    --callbacks http://127.0.0.1:5555/callback # Mandatory for this example, don't change.
```

Reply:
Good point, I missed that. Thanks!